### PR TITLE
Remove gnome-software and plasma-discover. Add Warehouse to yafti system apps

### PR DESCRIPTION
**Pull request by spaceoden**
#490

### CODE REVIEW

File: recipes/common/silverblue-modules.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ modules:
         - malcontent-ui-libs
         - malcontent-control
         - fedora-chromium-config-gnome
+        - gnome-software
```
Comment (RoyalOughtness) [inline recipes/common/silverblue-modules.yml:23]: also remove epiphany-runtime as it's no longer needed

File: recipes/common/kinoite-modules.yml [hunk 2]
```diff
@@ -8,6 +8,7 @@ modules:
       - fuse-encfs
       - krfb
       - krfb-libs
+      - plasma-discover
```
Comment (RoyalOughtness) [inline recipes/common/kinoite-modules.yml:11]: also remove plasma-discover-rpm-ostree

File: recipes/common/silverblue-modules.yml (hunk 1 above)
Comment (RoyalOughtness) [reply recipes/common/silverblue-modules.yml:23]: and gnome-software-rpm-ostree

**Comment by spaceoden** [comment]
Added the rpm-ostree and epiphany packages. thanks

**Pull request merged**